server/src/main/java/io/druid/server/http/DatasourcesResource.java

```diff
@@ -200,6 +201,53 @@ public Response deleteDataSource(
     return Response.ok().build();
   }
 
+  /*
+  Uncomment this method once the method deleteSourceName (just above this) is deleted.
```

Comment:
@fjy the existing endpoint is still present see [here](https://github.com/druid-io/druid/pull/2276/files#diff-9ad19c35d703fbc035ab9725dcbb9fafR173). This commented endpoint was supposed to replace the existing one but cannot until the previous one is removed completely form the code as both corresponds to same path - `@Path("/{dataSourceName}")`
@harshjain2 feel free to correct if I am wrong